servers/audio/audio_stream.h

```diff
@@ -49,6 +49,7 @@ class AudioSamplePlayback : public RefCounted {
 public:
 	Ref<AudioStream> stream;
 
+	Node *player = nullptr;
```

Comment:
Node pointers can get invalid if nodes are freed. Isn't there a risk of this happening here, and the audio driver would crash?

I would recommend storing an ObjectID instead, and getting the validated object instance from the ID.